### DISCUSSION TITLE
How not to mark links starting with a specific character

### DISCUSSION THREAD

**Discussion by danyliukdmitry** [ueberdosis/tiptap#2606] [General Questions]
<img width="156" alt="image" src="https://user-images.githubusercontent.com/99971458/157534919-1a955ff0-fa0c-4d4f-9d68-d6057c4584d6.png">

I need to ignore and untag links starting with @ and #

**Comment by philippkuehn** [comment]
Hmm, I think it’s not that easy to change the default behavior here. You can try to disable the [autolink option](https://tiptap.dev/api/marks/link#autolink).

Another solution is to track the changes, search for links starting with `@` and remove the mark manually.
